src/lib/agent/PocAgent.ts

```
@@ -0,0 +1,522 @@
+import { createInteractionTool } from '@/lib/tools/navigation/InteractionTool';
+import { createScrollTool } from '@/lib/tools/navigation/ScrollTool';
+import { createSearchTool } from '@/lib/tools/navigation/SearchTool';
+import { createRefreshStateTool } from '@/lib/tools/navigation/RefreshStateTool';
+import { createTabOperationsTool } from '@/lib/tools/tab/TabOperationsTool';
+import { createGroupTabsTool } from '@/lib/tools/tab/GroupTabsTool';
+import { createGetSelectedTabsTool } from '@/lib/tools/tab/GetSelectedTabsTool';
+import { createValidatorTool } from '@/lib/tools/validation/ValidatorTool';
+import { createScreenshotTool } from '@/lib/tools/utils/ScreenshotTool';
+import { createExtractTool } from '@/lib/tools/extraction/ExtractTool';
+import { createResultTool } from '@/lib/tools/result/ResultTool';
+import { createSubAgentTool } from '@/lib/tools/agent/SubAgentTool';
```

ESLint

[error] 'createSubAgentTool' is defined but never used.

(@typescript-eslint/no-unused-vars)